python/lsst/daf/butler/dimensions/_group.py

```diff
@@ -449,6 +450,24 @@ def add_to_order(element: DimensionElement) -> None:
         order.extend(element for element in self.elements if element not in done)
         return tuple(order)
 
+    @property
+    def region_dim(self) -> TopologicalRelationshipEndpoint | None:
```

Comment:
I'm not a huge fan of "dim" here as I don't think we've used that abbreviation (outside of local variable names) before, but I don't have anything better.  Technically it's a `DimensionElement` that's returned, but `region_dimension_element` is pretty verbose and `region_element` would expect the user to know what "element" in this context, just like `region_endpoint` would expect them to understand `TopologicalRelationshipEndpoint` (even worse).  Maybe `region_provider`?  Up to you; I clearly don't have any solutions.

I do think it might be good to switch this to return the `.name` attribute (i.e. a `str`) for consistency with the rest of the `DimensionGroup` API, and our overall lean towards minimizing appearance of the `DimensionElement` objects in the public API.

Reply:
Adopted `region_dimension(self) -> str`